source/common/ssl/context_impl.cc

```diff
@@ -544,8 +549,15 @@ void ServerContextImpl::updateConnectionContext(SSL* ssl) {
   ASSERT(SSL_CTX_get_ex_data(ctx_.get(), sslContextIndex()) == this);
 
   // Update SSL-level settings and parameters that are inherited from SSL_CTX during SSL_new().
+  // TODO(PiotrSikora): add SSL_early_set_SSL_CTX() to BoringSSL.
+
+  // TODO(PiotrSikora): add getters to BoringSSL.
+  SSL_set_min_proto_version(ssl, min_protocol_version_);
```

Comment:
Please RELEASE_ASSERT success on these two lines, to be consistent with the same calls in ContextImpl constructor.

Reply:
Good catch, thanks! I've added `ASSERT()` instead, since those values (and therefore successful return code) are already verified to work when creating original `SSL_CTX` in `ContextImpl:: ContextImpl()`.